VTEX - VTEX Do API.json

```diff
@@ -916,6 +882,19 @@
                 "in": "header",
                 "name": "X-VTEX-API-AppToken"
             }
+        },
+        "parameters": {
+            "reason": {
+                "name": "reason",
+                "in": "query",
+                "description": "This parameter is relevant only for PII-compliant accounts. When sending requests to this endpoint, PII-compliant accounts can use this parameter to declare the reason for requesting unmasked data. Otherwise this endpoint will return masked PII data.",
```

Comment:
```suggestion
                "description": "This parameter is relevant only for PII-compliant accounts. When sending requests to this endpoint, PII-compliant accounts can use this parameter to declare the reason for requesting unmasked data. Otherwise, this endpoint will return masked PII data.",
```